src/chess/chess.js

```diff
@@ -33,27 +33,26 @@
  * https://github.com/jhlywa/chess.js/blob/master/LICENSE
  */
 
-var Chess = function(fen, variant=0) {
-
+const Chess = function (fen) {
```

Comment:
You shouldn't have do npm run lint to this file, I think the linting causes the unit & integration test failing. We should put this in .eslintignore and change back to whatever syntax it was

Reply:
got you